db/views/rejection_percentage_by_days_v01.sql

```diff
@@ -0,0 +1,69 @@
+WITH rejection_reasons AS (
```

Comment:
So yes it does not a lot cleaner but I'm really not convinced it's helpful on such queries, In fact i did do that at first but when debugging the query I found my self jumping up and down the file and loosing the focus off the main query. 
I have very mixed feelings about this syntax specially since we are not reusing those bits anywhere which is the main reason behind them.
I'm a happy to change it but be aware of that.